gateway/src/kaazing/gateway/internal/wsn/WebSocketNativeDelegateHandler.js

```diff
@@ -84,10 +81,7 @@ var WebSocketNativeDelegateHandler = (function () {
             channel._delegate.setIdleTimeout(timeout);
         } catch (e) {
             // ignore exceptions thrown by older WebSocket impls in browsers
-            ;
```

Comment:
Do we know how these got inserted into the code originally?  Please identify the commit that introduced these changes.  In other words "Which commit added ; on separate lines for LOG statements?"

We need to grep the code and fix all locations where this is occurring, as it will introduce a performance penalty with the non-debug version of the javascript code.